Makefile

```diff
@@ -2,10 +2,12 @@
 RM := rm -f
 SSH_USER := core
 TERRAFORM_INSTALLER_URL := github.com/dcos/terraform-dcos
-DCOS_VERSION := 1.12
-CUSTOM_DCOS_DOWNLOAD_PATH := https://downloads.dcos.io/dcos/stable/1.12.0/dcos_generate_config.sh
-KUBERNETES_VERSION ?= 1.12.2
-KUBERNETES_FRAMEWORK_VERSION ?= 2.0.1-1.12.2
+DCOS_CLI_VERSION := 1.12
```

Comment:
Why the change in name from DCOS_VERSION to DCOS_CLI_VERSION?

Reply:
This is actually only used when downloading the CLI, `CUSTOM_DCOS_DOWNLOAD_PATH` is where the DCOS version is defined.

Reply:
Ahh just making it clearer, ok, fair.